src/WPML_OptionsManager.php

```diff
@@ -547,59 +563,55 @@ private function displayMP3Banner() {
             $option = rand(1, 2);
             add_option('wpml_banner_version', $option);
         }
-        ?>
-        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;margin-top: 15px;padding: 20px;">
-            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
-            <?php
-              if ($option === 1) {
-                    $this->displayMP3BannerVersionA();
-              } else {
-                    $this->displayMP3BannerVersionB();
-              }
-            ?>
-        </div>
-        <?php
+
+        $banner = '<div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;margin-top: 15px;padding: 20px;">';
+        $banner .= sprintf('<h3>%s</h3>', __( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' ) );
+
+        if ($option === 1) {
+            $banner .= $this->displayMP3BannerVersionA();
+		} else {
+			$banner .= $this->displayMP3BannerVersionB();
+		}
+		
+		$banner .= '</div>';
+
+		return $banner;
     }
 
     private function displayMP3BannerVersionA() {
-        ?>
-            <ul style="list-style-type:disc;list-style-position: inside">
-                <li><?php _e( '50 email templates to choose from', 'wp-mail-logging' );?></li>
-                <li><?php _e( 'Fun email designer', 'wp-mail-logging' );?></li>
-                <li><?php _e( 'Automated email marketing', 'wp-mail-logging' );?></li>
-                <li><?php _e( 'WooCommerce emails', 'wp-mail-logging' );?></li>
-                <li><?php _e( '99.1% success email deliverability', 'wp-mail-logging' );?></li>
-                <li><?php _e( 'Fast and friendly support', 'wp-mail-logging' );?></li>
-                <li><?php _e( 'Over 100,000 active users', 'wp-mail-logging' );?></li>
-            </ul>
-            <a
-                class="button button-primary"
-                href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_1"
-                target="_blank"
-            >
-                <?php _e( 'Try MailPoet for free', 'wp-mail-logging' );?>
-            </a>
-            <p>
-                <?php _e( 'Testimonial', 'wp-mail-logging' );?>:
-                <i>
-                    <?php _e( 'Thanks for this awesome plugin, love love love how it integrates with WordPress. I seriously spent days if not weeks on Mailchimp, and still haven’t been able to do what I did on MailPoet in 1 hour!', 'wp-mail-logging' );?>
-                </i>
-                — Kida Shey
-            </p>
-        <?php
+		$banner = '<ul style="list-style-type:disc;list-style-position: inside">';
+		$banner .= sprintf( '<li>%s</li>', __( '50 email templates to choose from', 'wp-mail-logging' ) );
```

Comment:
These changes use tab-based indentation, while the rest of the project uses space-based indentation.
Could you please use space-based indentation as well, matching the coding style of the project?